frontend/src/pages/profile/tokens.js

```diff
@@ -22,62 +21,24 @@ import { getSpinnerRow } from '../../utilities';
 import { FilterTable } from '../../components/filtertable';
 import AddTokenModal from '../../components/add-token-modal';
 import DeleteModal from '../../components/delete-modal';
-import ToastWrapper from '../../components/toast-wrapper';
 import { ALERT_TIMEOUT } from '../../constants';
 
+const COLUMNS = ['Name', 'Token', 'Expires', ''];
 
-export class UserTokens extends React.Component {
+const UserTokens = () => {
 
-  static propTypes = {
-    location: PropTypes.object,
-    navigate: PropTypes.func,
-    eventEmitter: PropTypes.object
-  };
+  const [tokens, setTokens] = useState([]);
```

Comment:
To fix loading state `const [tokens, setTokens] = useState([getSpinnerRow(4)]);`

Reply:
changing the logic, I moved this into the JSX so that `tokens` doesn't hold a list of not tokens.